src/system/SecurityCenterModule/Listener/FilterListener.php

```diff
@@ -224,6 +232,7 @@ public function idsInputFilter(RequestEvent $event): void
      */
     private function getIdsConfig(): array
     {
+        $vendorDir = $this->kernel->getProjectDir() . '/vendor/';
```

Comment:
probably could inject only the `project_dir` instead of the full kernel.

Reply:
Which syntax would that use?

Reply:
there are many examples in the core

Reply:
https://github.com/zikula/core/blob/50cacd745ac12e9c81bedda7a6193e130194b02b/src/Zikula/CoreBundle/Resources/config/services.yaml#L8

Reply:
updated